packages/blade/patches/react-native+0.66.5.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp b/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp
+index 2c68674..ea95e40 100644
+--- a/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp
++++ b/node_modules/react-native/ReactCommon/yoga/yoga/Yoga.cpp
+@@ -2229,7 +2229,7 @@ static float YGDistributeFreeSpaceSecondPass(
+         depth,
+         generationCount);
+     node->setLayoutHadOverflow(
+-        node->getLayout().hadOverflow() |
```

Comment:
do we have an upstream issue tracking this bug in react-native? This seems like a bug in RN itself. 

Or is it just related to our setup?

Reply:
This has been fixed in recent react-native release so once we upgrade we won't need this patch